pkg/apis/crds/karpenter.sh_machines.yaml

```diff
@@ -24,15 +24,27 @@ spec:
     - jsonPath: .metadata.labels.topology\.kubernetes\.io/zone
       name: Zone
       type: string
-    - jsonPath: .metadata.labels.karpenter\.sh/capacity-type
-      name: Capacity
+    - jsonPath: .status.nodeName
+      name: Node
       type: string
     - jsonPath: .status.conditions[?(@.type=="Ready")].status
       name: Ready
       type: string
     - jsonPath: .metadata.creationTimestamp
       name: Age
       type: date
+    - jsonPath: .metadata.labels.karpenter\.sh/capacity-type
+      name: Capacity
+      priority: 1
+      type: string
+    - jsonPath: .metadata.labels.karpenter\.sh/provisioner-name
+      name: Provisioner
+      priority: 1
```

Comment:
Is this the same mechanism as website doc priority? Like the higher the priority, the higher up it is? If so, my opinion is that it should be in order of 
1) node name
2) provisioner name
3) machineTemplateRef name
4) capacity type
(...) whatever else

Reply:
No, it's opposite. These are strictly ordered, so the output looks like

```
➜  karpenter git:(machine-migration) ✗ k get machines
NAME                             TYPE        ZONE         NODE                                         READY   AGE
trackgentle-2-alytu2h2ig-jqvzc   m3.medium   us-west-2c   ip-172-31-3-144.us-west-2.compute.internal   False   61s
➜  karpenter git:(machine-migration) ✗ k get machines -o wide
NAME                             TYPE        ZONE         NODE                                         READY   AGE   CAPACITY    PROVISIONER                TEMPLATE
trackgentle-2-alytu2h2ig-jqvzc   m3.medium   us-west-2c   ip-172-31-3-144.us-west-2.compute.internal   True    68s   on-demand   trackgentle-2-alytu2h2ig   sightregal-1-kd3mqecoec
```

Reply:
Out of curiosity, why do you prefer the ordering you listed above?

Reply:
That's funny, `priority` seems backwards to me then. Oh well! Thanks for the info!

Anyway, I think for printer columns, what you have looks good to me!